src/BrowserFetcher.js

```diff
@@ -32,12 +32,14 @@ const getProxyForUrl = require('proxy-from-env').getProxyForUrl;
 const downloadURLs = {
   chrome: {
     linux: '%s/chromium-browser-snapshots/Linux_x64/%d/%s.zip',
+    freebsd: '%s/chromium-browser-snapshots/Linux_x64/%d/%s.zip',
```

Comment:
This and the Firefox URL below are the same ones as for Linux? In that case let's simplify the patch by removing this line and line 42 and treating `'freebsd'` as equivalent to `'linux'`.

That way this change also doesn't impact the public `puppeteer.createBrowserFetcher` API.